### DISCUSSION TITLE
高亮视频、专栏标题中的 av、BV、cv 等一大堆乱七八糟号和用户艾特（@）

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#2071] [功能建议 / Ideas]
受著名 GitHub 插件 Refined GitHub 的 [linkify-code](https://github.com/sindresorhus/refined-github/blob/main/source/features/linkify-code.tsx) 功能启发。

在视频播放页和专栏页中高亮视频、专栏标题中的 av、BV、cv 等一大堆乱七八糟号和用户艾特（@）。

不过最后一个是不是不太可能？

**Comment by the1812** [comment]
标题中的? 感觉很少看见啊

**Reply by Geeyun-JY3** [reply]
确实不多，纯属突发奇想（受  RGH 启发）。可能艾特多一点？

**Reply by Geeyun-JY3** [reply]
所以也挺适合放到 2.0 之后做（
